lib/setup_helpers.sh

```diff
@@ -4,6 +4,11 @@
 VERSION_PARSER='use version; my ($version) = $_ =~ /([0-9]+([.][0-9]+)+)/; if (version->parse($version) lt version->parse($min)) { exit 1 };'
 CURL_RETRY_OPTIONS='--connect-timeout 5 --max-time 10 --retry 5 --retry-delay 0 --retry-max-time 60'
 
+KIT_PATH=$(dirname "$0")
+. "$KIT_PATH/../enterprise.constants"
+
+PROTOCOL=$GITHUB_PROTOCOL
```

Comment:
It would not be necessary, but I was thinking that the helpers normaly are independent from GitHub, so I wanted to avoid to mention in the helper functions no `GITHUB_*` variable.
What would you prefer?